ts/components/services/ServiceList.tsx

```diff
@@ -15,38 +28,128 @@ type AnimatedProps = {
 };
 
 type OwnProps = {
+  // Can't use ReadonlyArray because of the SectionList section prop typescript definition.
+  // tslint:disable-next-line:readonly-array
+  sections: Array<SectionListData<pot.Pot<ServicePublic, Error>>>;
+  profile: ProfileState;
+  isRefreshing: boolean;
+  onRefresh: () => void;
+  onSelect: (service: ServicePublic) => void;
+  readServices: ReadStateByServicesId;
+  isExperimentalFeaturesEnabled?: boolean;
   ListEmptyComponent?: React.ComponentProps<
-    typeof FlatList
+    typeof SectionList
   >["ListEmptyComponent"];
 };
 
 type Props = OwnProps & AnimatedProps;
 
-const AnimatedFlatList = Animated.createAnimatedComponent(FlatList);
+const styles = StyleSheet.create({
+  listItem: {
+    paddingLeft: variables.contentPadding,
+    paddingRight: variables.contentPadding
+  },
+  padded: {
+    marginLeft: customVariables.contentPadding,
+    marginRight: customVariables.contentPadding
+  }
+});
+
+const AnimatedSectionList = Animated.createAnimatedComponent(SectionList);
 
+/**
+ * A component to render a list of services grouped by organization.
+ */
 class ServiceList extends React.Component<Props> {
-  private flatListRef = React.createRef<typeof AnimatedFlatList>();
+  private sectionListRef = React.createRef<typeof AnimatedSectionList>();
+
+  private renderServiceItem = (
+    itemInfo: ListRenderItemInfo<pot.Pot<ServicePublic, Error>>
+  ) => (
+    <NewServiceListItem
+      item={itemInfo.item}
+      profile={this.props.profile}
+      onSelect={this.props.onSelect}
+      isRead={this.isRead(itemInfo.item, this.props.readServices)}
+      hideSeparator={true}
+    />
+  );
+
+  private getServiceKey = (
+    potService: pot.Pot<ServicePublic, Error>,
+    index: number
+  ): string => {
+    return pot.getOrElse(
+      pot.map(
+        potService,
+        service => `${service.service_id}-${service.version || 0}`
+      ),
+      `service-pot-${index}`
+    );
+  };
 
-  private renderItem = () => {
-    return <View />;
+  private renderServiceSectionHeader = (info: {
+    section: SectionListData<pot.Pot<ServicePublic, Error>>;
```

Comment:
here you haven't properly a **_pot.Pot<ServicePublic, Error>_** item.
This is because in selector you defined in `ts/store/reducers/entities/services/index.ts` (line 64) you build an object with these properties
![Schermata 2019-08-19 alle 13 08 09](https://user-images.githubusercontent.com/822471/63260890-71a9c800-c282-11e9-8e54-81fda218ac53.png)

So I think it's preferred to define an explicit type fo selector returned data, something like this:
```typescript
type OrganizationDataType = {
  organizationName: string;
  organizationFiscalCode: string;
  data: ReadonlyArray<pot.Pot<ServicePublic, Error>>;
};
```
What do think?